example/unicorn/components/crispy.py

```diff
@@ -0,0 +1,55 @@
+from django import forms
+
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import Submit
+
+from django_unicorn.components import UnicornView
+
+
+class ExampleForm(forms.Form):
+    like_website = forms.TypedChoiceField(
+        label="Do you like this website?",
+        choices=((1, "Yes"), (0, "No")),
+        coerce=lambda x: bool(int(x)),
+        widget=forms.RadioSelect,
+        initial="1",
+        required=True,
+    )
+
+    favorite_food = forms.CharField(
+        label="What is your favorite food?",
+        max_length=80,
+        required=True,
+    )
+
+    favorite_color = forms.CharField(
+        label="What is your favorite color?",
+        max_length=80,
+        required=True,
+    )
+
+    favorite_number = forms.IntegerField(
+        label="Favorite number",
+        required=False,
+    )
+
+    notes = forms.CharField(
+        label="Additional notes or feedback",
+        required=False,
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.helper = FormHelper()
+        self.helper.form_id = "id-exampleForm"
+        self.helper.form_class = "blueForms"
```

Comment:
you can remove this line from the unicorn crispy example as well - this is just a CSS that is added, from the crispy example.